content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -118,81 +118,140 @@ On your validator:
 
-To connect your validator to the network using public hubs, set the following configurations in your validator’s `rippled.cfg` file.
+Your validator needs to connect to the public hub once and only for a short amount of time to find other peers. After doing so, your server may or may not remain connected to the hub, depending on how stable your network connection is, how busy the hub is, and how many other high-quality peers your server finds. Your validator saves the addresses of these other peers so it can try reconnecting directly to those peers later, even if you restart your validator.
 
-1. Include the following `[ips_fixed]` stanza. The two values, `r.ripple.com 51235` and `zaphod.alloy.ee 51235`, are public hubs. This stanza tells `rippled` to always attempt to maintain peer connections with these public hubs.
+**Pros:**
 
-        [ips_fixed]
-        r.ripple.com 51235
-        zaphod.alloy.ee 51235
+  - Increases the chances of your validator finding a peer, probably several, to connect to.
 
-    You can include the IP addresses of other `rippled` servers here, but _**only**_ if you can expect them to:
+  - Creates the opportunity for a lot of direct peer connections. Being able to request content from more peers enables your validator to parallelize the process of downloading the current ledger (when syncing) or backfilling history and gives it direct access to a wider selection of history.
 
-      - Relay messages without censoring.
-      - Stay online consistently.
-      - Not DDoS you.
-      - Not try to crash your server.
-      - Not publish your IP address to strangers.
+  - Lowers the possibility of your validator disconnecting from the network because it provides the ability to constantly replace disconnected peers with new ones.
 
-2. Include the following `[peer_private]` stanza and set it to `1`. Enabling this setting instructs your validator’s peers not to broadcast your validator’s IP address. This setting also instructs your validator to connect to only the peers configured in your `[ips_fixed]` stanza. This ensures that your validator connects to and shares its IP with only peer `rippled` servers you know and trust.
+**Cons:**
 
-    **Warning:** Be sure that you don't publish your validator's IP address in other ways.
+  - Doesn't allow you to select your validator's peers, which means that you have no idea whether your peers may decide to act maliciously.
 
-        [peer_private]
-        1
+    If your peers do act maliciously, your server has some protections that it can use to ignore or block them. Precautions include marking peers as `insane`, tracking resource usage, blocking peers that request too much stuff, and verifying signatures. Also, one of the server's protections is simply being designed not to crash no matter what your peers send you.
```

Comment:
@mellery451 - thank you for asking! It is not a separate "con," but rather a clarification of how to avoid the malicious peer "con." I was hoping that making it a sub-paragraph (with no bullet) of the bulleted "con" would convey this - but perhaps it's too subtle? 🤔